docker-compose.yml

```diff
@@ -6,12 +6,16 @@ services:
     restart: unless-stopped
     ports:
       - 3307:3306
+    volumes:
+      - mysql-data:/var/lib/mysql/
   psql:
     env_file: [.env_file]
     image: postgres:latest
     restart: unless-stopped
     ports:
       - 5432:5432
+    volumes:
+      - pg-data:/var/lib/postgresql/data/
```

Comment:
So, are sure that we need local volumes (that AFAIK are supposed to be for development) for these two databases only?

Reply:
It's legacy from older dev